docs/src/main/sphinx/security/internal-communication.rst

```diff
@@ -25,107 +26,36 @@ command:
 Internal SSL/TLS configuration
 ------------------------------
 
-SSL/TLS is configured in the ``config.properties`` file.  The SSL/TLS on the
-worker and coordinator nodes are configured using the same set of properties.
-Every node in the cluster must be configured. Nodes that have not been
-configured, or are configured incorrectly, are not able to communicate with
-other nodes in the cluster.
+The SSL/TLS on the worker and coordinator nodes are configured using the same
+set of properties. Every node in the cluster must be configured. Nodes that have
+not been configured, or are configured incorrectly, are not able to communicate
+with other nodes in the cluster.
 
-To enable SSL/TLS for Trino internal communication, do the following:
+You must :doc:`/security/tls` for access to the cluster, before enabling SSL/TLS
```

Comment:
I don't think this is true.  I'm pretty sure you can have internal on HTTPS and users on raw HTTP, but I didn't test it.

Reply:
> BTW the old keystore stuff still works if people want to do that, but I don't think we need that level of detail.

sure.. and if they want to use the old complex way .. they should look at the old docs.. I dont think we should keep that info.

Reply:
I realize that this is probably possible.. but it makes no sense whatsoever to do that ..  first you have to have your access to the cluster secure ... without that there is no sense in encrypting the traffic inside the cluster.

Reply:
We chatted on slack and I softened the language a bit..